diff --git a/dwm.c b/dwm.c
index f1d86b2..82661de 100644
--- a/dwm.c
+++ b/dwm.c
@@ -708,7 +708,7 @@ drawbar(Monitor *m)
 		return;
 
 	/* draw status first so it can be overdrawn by tags later */
-	if (m == selmon) { /* status is only drawn on selected monitor */
+	if (m == selmon || 1) { /* status is only drawn on selected monitor */
 		char *text, *s, ch;
 		drw_setscheme(drw, scheme[SchemeNorm]);
 
@@ -2084,6 +2084,7 @@
 void
 updatestatus(void)
 {
+	Monitor* m;
 	if (!gettextprop(root, XA_WM_NAME, stext, sizeof(stext))) {
 		strcpy(stext, "dwm-"VERSION);
 		statusw = TEXTW(stext) - lrpad + 2;
@@ -2101,7 +2102,8 @@
 		}
 		statusw += TEXTW(text) - lrpad + 2;
 	}
-	drawbar(selmon);
+	for(m = mons; m; m = m->next)
+		drawbar(selmon);
 }

 void

